templates/javascript/rewrite_links_html.js

```diff
@@ -21,11 +18,9 @@ class AttributeRewriter {
     }
   }
 }
-
 const rewriter = new HTMLRewriter()
   .on('a', new AttributeRewriter('href'))
   .on('img', new AttributeRewriter('src'))
-
```

Comment:
it's unfortunate the TypeScript compiler strips out these newlines from the resulting JS output. we should consider using Babel to strip type annotations instead of `tsc`